resonantLimits/run_limits.py

```diff
@@ -94,28 +94,28 @@ def run_limits(in_tags, channels, selections, selection_prefixes, masses,
     
     channels = ('ETau', 'MuTau', 'TauTau')
     tag_ = '23Mar'
-    suffix = '_mHH'
+    suffix = ''
     #in_tags = ['{}_{}_{}{}'.format(tag_, x, period, suffix) for x in channels]
     in_tags = ['{}_{}{}'.format(tag_, x, suffix) for x in channels]
     cfg_files = ['mainCfg_{}_{}{}.cfg'.format(x, period, suffix)
                  for x in channels]
     out_tag = '{}_{}{}'.format(tag_, period, suffix)
-    # selections = ('s1b1jresolvedMcut', 's2b0jresolvedMcut', 'sboostedLLMcut')
-    selections = ('s1b1jresolvedMcutmHH250_335',
-                  's1b1jresolvedMcutmHH335_475',
-                  's1b1jresolvedMcutmHH475_725',
-                  's1b1jresolvedMcutmHH725_1100',
-                  's1b1jresolvedMcutmHH1100_3500',
-                  's2b0jresolvedMcutmHH250_335',
-                  's2b0jresolvedMcutmHH335_475',
-                  's2b0jresolvedMcutmHH475_725',
-                  's2b0jresolvedMcutmHH725_1100',
-                  's2b0jresolvedMcutmHH1100_3500',
-                  'sboostedLLMcutmHH250_625',
-                  'sboostedLLMcutmHH625_775',
-                  'sboostedLLMcutmHH775_1100',
-                  'sboostedLLMcutmHH1100_3500'
-                  )
+    selections = ('s1b1jresolvedMcut', 's2b0jresolvedMcut', 'sboostedLLMcut')
+    # selections = ('s1b1jresolvedMcutmHH250_335',
+    #               's1b1jresolvedMcutmHH335_475',
+    #               's1b1jresolvedMcutmHH475_725',
+    #               's1b1jresolvedMcutmHH725_1100',
+    #               's1b1jresolvedMcutmHH1100_3500',
+    #               's2b0jresolvedMcutmHH250_335',
+    #               's2b0jresolvedMcutmHH335_475',
+    #               's2b0jresolvedMcutmHH475_725',
+    #               's2b0jresolvedMcutmHH725_1100',
+    #               's2b0jresolvedMcutmHH1100_3500',
+    #               'sboostedLLMcutmHH250_625',
+    #               'sboostedLLMcutmHH625_775',
+    #               'sboostedLLMcutmHH775_1100',
+    #               'sboostedLLMcutmHH1100_3500'
+    #               )
     selection_prefixes = ('s1b1jresolvedMcut', 's2b0jresolvedMcut', 'sboostedLLMcut')
     
     #masses = ('250', '260', '280', '300', '320', '350', '400', '450', '500', '550', '600', '650', '700', '750', '800', '850', '900', '1000', '1250', '1500', '1750', '2000', '2500', '3000')
```

Comment:
I do not think in this script you wanted to comment every step apart from limits, can you add back the previous version?